bin/generate.dart

```diff
@@ -275,18 +275,56 @@ class DiagramGenerator {
       List<AnimationMetadata> metadataList) async {
     final Directory destDir = Directory(assetDir);
     final List<File> outputs = <File>[];
-    final List<WorkerJob> jobs = <WorkerJob>[];
+    // The key represent the iterations and starts with 0. Iterations are
+    // executed sequentially.
+    //
+    // This can be useful if one job relies on the product of another job.
+    // TODO(chunhtai): figure out how to `join` WorkJob[s] so that this iteration
+    // logic is not needed.
+    final Map<int, List<WorkerJob>> jobs = <int, List<WorkerJob>>{};
```

Comment:
Not sure if there is a better way to make one workerJob to wait for another since the ProcessPool runs everything in parallel.